settings.gradle

```diff
@@ -14,8 +14,9 @@ pluginManagement {
 
 rootProject.name = 'Artemis'
 
+// needed for rest call and endpoint analysis
 include 'supporting_scripts:analysis-of-endpoint-connections'
 
 // needed for programming exercise templates
 DirectoryScanner.removeDefaultExclude "**/.gitattributes"
-DirectoryScanner.removeDefaultExclude "**/.gitignore"
```

Comment:
Removing the new line at the end of a file violates the [editorconfig](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#insert_final_newline) of Artemis. Please reinsert the newline here.

https://github.com/ls1intum/Artemis/blob/740366a7c4725f6b53ad9c3b8ca26682db9ea6cf/.editorconfig#L13

Reply:
In my commit, there is a new line at the end. In the original settings.gradle, there is no final new line. The new line was added by the Artemis editorconfig. I will remove it again to keep it in its original state.